detection-rules/download_link_to_disk_image_in_encrypted_zip.yml

```diff
@@ -0,0 +1,25 @@
+name: "Download link to disk image in encrypted zip"
+description: |
+  A link in the body of the email downloads an encrypted zip that contains a disk image of the format IMG, ISO or VHD. This is a combination of file types used to deliver Qakbot.
+type: "rule"
+references:
+  - "https://twitter.com/pr0xylife/status/1592502966409654272"
+severity: "medium"
+authors:
+  - twitter: "ajpc500"
+source: |
+  type.inbound and 
+  any(body.links, 
+      any(beta.linkanalysis(.).files_downloaded, 
+          any(
+              beta.binexplode(.), (
+                  any(.flavors.yara, . == "encrypted_zip") and
+                  any(.scan.zip.all_paths, any([".img", ".iso", ".vhd"], strings.ends_with(.., .)))
+              )
+          )
+      )
+  )

```

Comment:
thoughts on adding the [FTS snippet](https://docs.sublimesecurity.com/docs/common-snippets#first-time-sender) at the end? since most qbot campaigns are spray and pray, i *think* that shouldn't make a big difference, but it might reduce analysis load + FPs?